lte/gateway/c/core/oai/tasks/grpc_service/SpgwServiceImpl.cpp

```diff
@@ -15,9 +15,9 @@
  *      contact@openairinterface.org
  */
 #include <string>
+#include <arpa/inet.h>
```

Comment:
not related to your changes but can you move `SpgwServiceImpl.h` to the top? that should be the first include header on this file

Reply:
Pushed.